db/migrate/20190110201340_remove_google_plus_from_share_links_site_setting.rb

```diff
@@ -8,6 +8,6 @@ def up
   end
 
   def down
-    raise ActiveRecord::IrreversibleMigration
+    # raise ActiveRecord::IrreversibleMigration
```

Comment:
I think this shouldn't be commented?

Reply:
Oops, I had to do that temporarily to re-build my database in dev. Fixed in an upcoming commit. Good catch.